wlib/SourceView.cpp

```diff
@@ -919,7 +932,18 @@ bool CSourceCtrl::StartAutoComplete()
         CString module(word, periodPos);
         if (m_type->IsTypeOf(ATTRIBUTE_TYPE_ECL))
         {
-            m_langRef->GetAutoC((const TCHAR *)module, list);
+            if (CComPtr<IEclCC> eclcc = CreateIEclCC())
+            {
+                if (boost::algorithm::starts_with(static_cast<const TCHAR *>(word),_T("$")))
```

Comment:
This logic should be inside the Meta Class as it "knows" more about ECL than the editor will.